ApiFirst/DomainServices/Services/CustomerServices.cs

```diff
@@ -1,19 +1,33 @@
-﻿namespace ApiFirst
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
 {
-    public class CustomerService : ICustomerService
+    public class CustomerServices : ICustomerServices
     {
         private readonly List<Customer> _customers = new();
 
         public bool Create(Customer model)
         {
-            model.Id = _customers.LastOrDefault()?.Id + 1 ?? 1;
+            model.Id = _customers.LastOrDefault()?.Id + 1 ?? 0;
 
             if (Exists(model)) return false;
 
```

Comment:
Ao invés de retornar um false você pode lançar exceções para serem tratadas na controller.
```suggestion

            if (Exists(model)) throw new ArgumentException($"User already exists with this email and cpf. Email: {model.Email}, Cpf: {model.Cpf});

```